script.js

```diff
@@ -1,13 +1,35 @@
 const [buttonsContainer] = document.getElementsByClassName("buttonsBlock");
 const [result] = document.getElementsByClassName("result");
+let calculating = [];
 
 const buttonClick = (event) => {
-  if (event.target.id === "clean") {
-    result.innerHTML = 0;
-    return;
+  switch (event.target.id) {
+    case "clean":
+      result.innerHTML = 0;
+      calculating = [];
+      break;
+    case "backspace":
+      calculating.pop();
+      if (calculating.length === 0) {
+        return (result.innerHTML = 0);
+      }
+      result.innerHTML = calculating.join("");
+      break;
+    case "square":
+      let number = calculating.join("");
+      result.innerHTML = Math.pow(number, 2);
+      calculating = [];
+      break;
+    case "partOfAWhole":
```

Comment:
Ideally, create object with constant (read Object.freeze). It will mean enums from TS